### PR TITLE
Resolve multiple URLs and process WET files (resolves issue#65)

### PR DESCRIPTION
**Pull request by nyxathid**
Added functionality to CC Index to resolve multiple URLs at once. Requests to the CC Index API are sent one by one. The API is very slow in general.

Two functions available:
- `getWETPathsForEachURL()` - callback called with an array of WET paths after all URLs were resolved
- `getWETPathsForEachURLStepByStep()` - same as above but with backup on disk after `n` requests

### CODE REVIEW

**Comment by nyxathid** [comment]
Added timeouts to all functions. Now you can run this even on 4k URLs and be sure that it will terminate after a while... a very long while...

**Comment by nyxathid** [comment]
Added processing WET files after resolving the URLs. The whole process takes very long: we have 4k URLs to resolve and probably about 2k+ WET files to process. Read the description to find out when you can kill the process and don't loose time & data.

**Comment by nyxathid** [comment]
Runs 80% faster now. Or maybe it's just my internet connection :D

File: src/cc-index.ts [hunk 1]
```diff
@@ -55,44 +57,227 @@ export class CCIndex {
...
+                                        callback : (wetPaths : Array<string>) => void,
+                                        timeout? : number,
+                                        ccIndexPageURL? : string) {
+
+        let wets : Set<string> = new Set<string>();
+        console.log("start looking up " + lookupURLs.length + " urls");
+
+        // We can't send 100500 requests at the same time, instead we:
+        //  - query cc index with one url,
+        //  - wait for the response,
```
Comment (lgfrbcsgo) [inline src/cc-index.ts]: Why though? Wouldn't it be much more efficient to have, let's say, 50 connections open at a time?

Comment (nyxathid) [inline src/cc-index.ts]: I tried to send a lot of request as the same time but it didn't work. But you're right, we should allow some parallelism.

**Comment by kordianbruck** [comment]
issue #65 
(Titles are not used for cross linking)

**Comment by nyxathid** [comment]
@lgfrbcsgo now I see your comment about `urls/previouslyResolvedWETs.json`. I committed it, so you can test the second step directly and don't have to wait for resolving. We can ignore it in future :+1:

**Pull request merged**